manticore/__main__.py

```diff
@@ -97,8 +97,8 @@ def ethereum_cli(args):
         print "[+] There are %d reverted states now"% len(m.terminated_state_ids)
         print "[+] There are %d alive states now"% len(m.running_state_ids)
 
-    for state in m.all_states:
-        print str(state.context['last_exception'])
+        for state in m.all_states:
```

Comment:
Will not this re-print the states from previous iterations ?

Reply:
yes it will, i'll rm this from the pr, it is not related and only for debugging